transport/http/constants.go

```diff
@@ -20,8 +20,12 @@
 
 package http
 
+import "time"
+
 const transportName = "http"
 
+var defaultConnTimeout = 500 * time.Millisecond
```

Comment:
I think we are, I've had comments on it from my code review bff @willhug. Note I don't like the underscore style, but yea.